Makefile

```diff
@@ -1,16 +1,17 @@
-all: build test sanity
-
-.PHONY: build
-build:
-	cargo build --release
+.PHONY: linux
+linux:
+	cargo build --release --target x86_64-unknown-linux-gnu
 	cargo install --path .
 	cargo clippy
 	cargo fmt
-
-.PHONY: test
```

Comment:
we should add this back. These are our unit tests and we want them to run in CI.